packages/presentation/lib/screen/home/widgets/homeSkelet.dart

```diff
@@ -0,0 +1,124 @@
+                                        color: Colors.yellow,
+                                      ),
+                                      Icon(
+                                        Icons.star_border,
+                                        color: Colors.yellow,
+                                      ),
+                                      Icon(
+                                        Icons.star_border,
+                                        color: Colors.yellow,
+                                      ),
+                                      Icon(
+                                        Icons.star_border,
+                                        color: Colors.yellow,
+                                      ),
+                                      Icon(
+                                        Icons.star_border,
+                                        color: Colors.yellow,
+                                      ),
+                                    ],
+                                  ),
+                                  Container(
+                                    height: 18,
+                                    color: Colors.white,
+                                  ),
+                                  const SizedBox(
+                                    height: 5,
+                                  ),
+                                  Row(
+                                    mainAxisAlignment: MainAxisAlignment.start,
+                                    children: [
+                                      Container(
+                                        height: 8,
+                                        width: 24,
+                                        color: Colors.white,
+                                      ),
+                                      const Icon(
+                                        Icons.circle,
+                                        size: 6,
+                                        color: Colors.grey,
+                                      ),
+                                      Container(
+                                        height: 8,
+                                        width: 44,
+                                        color: Colors.white,
+                                      ),
+                                      const SizedBox(
+                                        width: 10,
+                                      ),
+                                      Container(
+                                        height: 8,
+                                        width: 24,
+                                        color: Colors.white,
+                                      ),
+                                    ],
+                                  )
+                                ],
+                              ),
+                            );
+                          },
+                        ),
+                        const Text('hi'),
```

Comment:
Seems that this string should be deleted, because there is no word 'hi' in design

Reply:
it's actually needed here, because tabbar view expects two widget to display(comming soon and now showing), but I fix it so it displays shimmer on both page instead of this stub